data/json/items/items_holiday.json

```diff
@@ -66,5 +66,46 @@
     "color": "green",
     "looks_like": "steel_plate",
     "use_action": { "type": "deploy_furn", "furn_type": "f_winter_wreath" }
+  },
+  {
+    "abstract": "inflatable_decoration",
+    "type": "TOOL",
+    "name": { "str": "inflatable" },
+    "description": "A generic inflatable abstract to use for specific items.",
+    "weight": "3 kg",
+    "volume": "5 L",
+    "price": 5000,
+    "material": [ "plastic" ],
+    "symbol": "P",
+    "color": "light_red",
+    "charges_per_use": 100,
+    "flags": [ "ALLOWS_REMOTE_USE" ]
+  },
+  {
+    "id": "inflatable_dragon",
+    "type": "TOOL",
+    "copy-from": "inflatable_decoration",
+    "name": { "str": "inflatable dragon" },
+    "description": "A battery powered inflatable dragon from the Halloween before the end.  It might be fun to display it somewhere.",
+    "pocket_data": [ { "pocket_type": "MAGAZINE", "ammo_restriction": { "battery": 500 } } ],
```

Comment:
Is it intentional that this counts as a battery itself instead of containing batteries?
Normally you'd specify "MAGAZINE_WELL" and one of the sizes of battery items.

Reply:
That would be better.  I'll make that change, thanks!